### ISSUE TITLE
What are Google Apps?

### ISSUE THREAD

**Issue by iteles** [nelsonic/google-apps-certification#4]
I'm not sure how versed the academics and others in our audience are in the concept of Google Apps.

I suggest it's worth adding a sentence at the beginning of the [Google Apps section](https://github.com/nelsonic/google-apps-certification/tree/c3d3e567f75a24da0023b7a4cbc6906ab2031248#google-apps) explaining what they are in layman's terms.

**Comment by nelsonic** [comment]
Please PR directly. :+1:

**Comment by iteles** [comment]
:+1:

**Comment by nelsonic** [comment]
see: https://github.com/nelsonic/google-apps-certification/blame/1ac77a30bcd85c7cd0180bf4a0514cef7138b55e/README.md#L37-L44

**Comment by iteles** [comment]
I actually think that the most important part here is of course the move from companies to Google Apps but also the fact that they are _web_ apps which means you can access them from anywhere with web access.

**Comment by iteles** [comment]
I re-wrote this to remove the references to Microsoft before I pushed because GA has a much wider reach than Microsoft products but was going to incorporate it back in with #5
